source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -80,6 +97,15 @@ content: |
            const patientSchema = {
              "medicalRecords.patients": jsonSchema,
            }
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+           :emphasize-lines: 2
+
+           patientSchema = {
```

Comment:
Would it be better to use snake case to stay consistent with other variables like `local_master_key`?